### PR TITLE
Fix android load error

### PR DESCRIPTION
**Pull request by Zen-CODE**
## Background

This PR fixes the below error. It also makes various pep8 fixes and moves kv loading to and external file (better tooling support/kv lang recognition).
```
 File "jnius/jnius_export_func.pxi", line 26, in jnius.find_javaclass
   File "jnius/jnius_jvm_dlopen.pxi", line 91, in jnius.create_jnienv
 jnius.JavaException: JVM exception occurred: android/os/Environment java.lang.NoClassDefFoundError
```

### CODE REVIEW

File: launcher/app.py [hunk 1]
```diff
@@ -2,174 +2,17 @@
 
 import os
 from datetime import datetime
-from kivy.lang import Builder
+from kivy.lang import Builder, builder
```
Comment (tshirtman) [inline launcher/app.py]: double import of Builder, and not sure why the `builder` module is imported as well.

**Comment by Zen-CODE** [comment]
@tshirtman Do we still want the circleci integration? Or switch to github?

**Comment by tshirtman** [comment]
I'd be of the opinion of switching to github actions, yes, the buildozer action makes that relatively easy, i have some examples if you need.

**Comment by dotrichard** [comment]
@tshirtman Bump. Can't see why CircleCI is failing and I'm not happy giving it full access to my personal and private repos -and it does not allow me to choose otherwise.

**Comment by tshirtman** [comment]
We can switch it to github actions, if you need help to do that i can have a look later today or this weekend.

**Comment by Zen-CODE** [comment]
That would help thanks. Otherwise I'll only get time next week to look at it.

**Comment by tshirtman** [comment]
https://github.com/kivy/kivy-launcher/actions/runs/772332600 :dancers: 

(btw, renamed the master `branch` to `main` don't be surprised)

**Review by tshirtman — APPROVED** [review]

**Pull request merged**